### PR TITLE
Make addthis floating tool bar works on desktop mode again

### PR DESCRIPTION
**Pull request by taojing10**

### CODE REVIEW

**Comment by torch2424** [comment]
cc @aghassemi Not entirely sure where this would be triaged to, so mentioning you 😂

File: extensions/amp-addthis/0.1/amp-addthis.js [hunk 1]
```diff
@@ -137,6 +139,9 @@ class AmpAddThis extends AMP.BaseElement {
     const pubId = this.element.getAttribute('data-pub-id') || '';
     const widgetId = this.element.getAttribute('data-widget-id') || '';
     const productCode = this.element.getAttribute('data-product-code') || '';
+    if (this.win.innerWidth >= 979){
```
Comment (aghassemi) [inline extensions/amp-addthis/0.1/amp-addthis.js]: Please use `this.getViewport().getWidth()`

Comment (taojing10) [reply extensions/amp-addthis/0.1/amp-addthis.js]: Fixed

Comment (taojing10) [reply extensions/amp-addthis/0.1/amp-addthis.js]: @aghassemi  do you know why `The command "node build-system/pr-check/checks.js" exited with 1.` this happened on continuous-integration/travis-ci/pr

**Comment by aghassemi** [comment]
@taojing10 LGTM. The CI failure is unrelated to your change. We are working on fixing it, I will merge when it is green.

**Comment by taojing10** [comment]
We had an issue on this. Will update it later

**Pull request closed without merging**